manifests/mod/event.pp

```diff
@@ -99,16 +99,9 @@
     notify  => Class['apache::service'],
   }
 
-  case $facts['os']['family'] {
-    'redhat': {
-      if versioncmp($_apache_version, '2.4') >= 0 {
-        apache::mpm { 'event':
-          apache_version => $_apache_version,
-        }
-      }
-    }
-    'debian', 'freebsd' : {
-      apache::mpm { 'event':
+  case $::osfamily {
+    'debian','freebsd','redhat' : {
```

Comment:
They need to be capitalized?

Reply:
don't think so

Reply:
Puppet does case insensitive comparisons on a case but the fact is capitalized. So if you change it to `if $::osfamily == 'redhat'` it will fail. That must be `if $::osfamily == 'RedHat`. Because of that I always try to keep the capitalization correct, even if it doesn't matter.